examples/terraform-submodules/gke/.gitignore

```
@@ -0,0 +1,2 @@
+development.tfvars
```

I didn't see anywhere in the docs suggesting these file names, so I'm not sure it's worth checking in a gitignore that may work for one developer but not for folks who choose different names for their custom variables.

---

This might be more useful:
```suggestion
*.tfvars
```
WDYT? We can have it here or on the repo level?